src/core/types.ts

```diff
@@ -114,6 +114,7 @@ export interface ExecutionOptions {
   updateRouteHook?: UpdateRouteHook
   updateTransactionRequestHook?: TransactionRequestUpdateHook
   executeInBackground?: boolean
+  disableMessageSigning?: boolean
```

Comment:
Since this is EVM specific for now, should we move this flag to be EVM executor only (EVMStepExecutor)?

Reply:
I thought about this; I think it looks cleaner having it in the `ExecutionOptions`; otherwise, we'd need to have this flag in the EVM's configuration and pass it down to the EVMStepExecutor.